spec/requests/api/v0/manifests_search_spec.rb

```diff
@@ -1,35 +1,31 @@
 require 'rails_helper'
-require_relative '../../db/chores/populator'
 require "queryparams"
 
 describe "search", elasticsearch: true do
+  before(:each) do
+    Manifest.rebuild_index
```

Comment:
I agree that this is similar to database cleaner, but it is not exactly the same. If I write a new test, I know that I will always want to a assume an empty DB. I am less sure that the state of the manifest index is relevant (but it might be!)

Will `Manifest.rebuild_index` be required for all `elasticsearch: true` specs? And if so, what about including it in that code somehow?